src/Common/ForceAsyncAwaiter.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.AspNetCore.Sockets.Internal
 {
-    public static class ForceAsyncTaskExtensions
+    internal static class ForceAsyncTaskExtensions
```

Comment:
I needed to make these internal because this file was linked by two projects, creating a conflict. Better solution?

Reply:
It should have been internal to begin with.